src/components/RecoveryWords.js

```diff
@@ -157,9 +165,10 @@ class RecoveryWords extends Nimiq.Observable {
     _setFocusToNextInput(index, paste) {
         index = Math.max(index, 0);
         if (index < this.$fields.length) {
-            this.$fields[index].focus();
-            if (paste) {
-                this.$fields[index].fillValueFrom(paste);
+            const field = this.$fields[index];
+            field.focus();
+            if (paste && 'fillValueFrom' in field) {
```

Comment:
```suggestion
            if (paste && 'fillValueFrom' in field && typeof field.fillValueFrom === 'function') {
```

not sure if this is relevant

Reply:
I don't think this is necessary.